sqlglot/dialects/hive.py

```diff
@@ -47,9 +47,31 @@
     "HOUR": " / 3600",
 }
 
+INTERVAL_VARS = {
+    "SECOND",
+    "SECONDS",
+    "MINUTE",
+    "MINUTES",
+    "DAY",
+    "DAYS",
+    "MONTH",
+    "MONTHS",
+    "YEAR",
+    "YEARS",
+}
+
+
 DIFF_MONTH_SWITCH = ("YEAR", "QUARTER", "MONTH")
 
 
+def _parse_number(self: Hive.Parser, token: TokenType) -> t.Optional[exp.Expression]:
+    number = parser.Parser.PRIMARY_PARSERS[TokenType.NUMBER](self, token)
```

Comment:
why not call super?

Reply:
because it's not in the context of a class definition and so python won't accept it

Reply:
super(...).x